### PR TITLE
BLOCKCHAIN-297 - Discussion links per proposal hash + links for all referenda + multiple links per proposal

### PR DESCRIPTION
**Pull request by kacperzuk-neti**

### CODE REVIEW

File: src/api/nodeRpcCall.js [hunk 1]
```diff
@@ -102,22 +102,20 @@ const getApi = async () => {
...
-    const centralizedData = allCentralizedData.find((cItem) => (
-      parseInt(cItem.chainIndex) == parseInt(item.index)
-    ))
-
     const proposalHash = isReferendum ? item.imageHash : (
       item.boundedCall?.Lookup?.hash_ ?? 
       item.boundedCall?.Legacy?.hash_
     )
+
+    const centralizedDatas = allCentralizedData.filter((cItem) => ( cItem.hash == proposalHash))
```
Comment (MichalLustykNeti) [inline src/api/nodeRpcCall.js:110]: can we use '===' instead of '=='?

File: src/api/backend.js [hunk 2]
```diff
@@ -57,10 +57,10 @@ export const getReferenda = async () => {
 };
 
 export const addReferendum = async ({
-  link, chainIndex, name, description, hash, additionalMetadata, proposerAddress,
+  link, name, description, hash, additionalMetadata, proposerAddress,
 }) => {
   const api = getApi();
-  return await api.post('/referenda', {
-    link, chainIndex, name, description, hash, additionalMetadata, proposerAddress,
+  return api.post('/referenda', {
```
Comment (MichalLustykNeti) [inline src/api/backend.js:63]: can we add await here?

Comment (kacperzuk-neti) [reply src/api/backend.js:63]: Why?

File: src/api/nodeRpcCall.js (hunk 1 above)
Comment (kacperzuk-neti) [reply src/api/nodeRpcCall.js]: Done.

**Pull request merged**